lua/wikis/commons/MatchTicker/DisplayComponents/New.lua

```diff
@@ -24,14 +23,15 @@ local VodLink = require('Module:VodLink')
 
 local DefaultMatchTickerDisplayComponents = Lua.import('Module:MatchTicker/DisplayComponents')
 local HighlightConditions = Lua.import('Module:HighlightConditions')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Title = Lua.import('Module:Widget/Tournament/Title')
 
 local OpponentLibraries = require('Module:OpponentLibraries')
 local Opponent = OpponentLibraries.Opponent
 local OpponentDisplay = OpponentLibraries.OpponentDisplay
 
 local CURRENT_PAGE = mw.title.getCurrentTitle().text
 local HIGHLIGHT_CLASS = 'tournament-highlighted-bg'
-local TOURNAMENT_DEFAULT_ICON = 'Generic_Tournament_icon.png'
```

Comment:
I opted to remove this from the matchticker, as the tournament ticker also wouldn't display it.